pkg/apis/v1alpha1/types.go

```diff
@@ -117,6 +117,8 @@ type CopyAnnotationsSpec struct {
 }
 
 type AuthSpec struct {
+	// Whether deployments should be run with "managed" service accounts created by the gateway controller.
+	Managed bool `json:"managed,omitempty"`
```

Comment:
Is the `Account` field still relevant if `Managed` is true? Should we validate that both are not set?

Reply:
I can submit a follow-up PR that documents a little bit better which takes priority, but no, both aren't still relevant. At the same time, I'm not so sure I'd want to consider the config invalid if both are specified, but I'm sure we can hash that out subsequently.